docs/README.md

```diff
@@ -55,32 +55,58 @@ Since Algolia doesn't allow filtering by arbitrary tags, we also added the
 as a "filter-only" facet.
 
 To actually filter by version during a search, we need to know which version of
-the docs the current page was built against.  We embed this information into
+the docs the current page was built against. We embed this information into
 the same `rel=root` link mentioned above.
 
 ### Sphinx-native search fallback
 
-Determined have some users whose clusters do not have internet access.  For
-those users Algolia's search-as-a-service model will never work.  When we
+Determined have some users whose clusters do not have internet access. For
+those users Algolia's search-as-a-service model will never work. When we
 detect that we are unable to load resources from aloglia, we fall back to the
-default Sphinx search.  See the JavaScript for implementation.
+default Sphinx search. See the JavaScript for implementation.
 
 ### Dev builds
 
 Because Algolia indexes after docs are published to docs.determined.ai,
-development builds of docs cannot rely on a versioned Algolia index.  Instead,
+development builds of docs cannot rely on a versioned Algolia index. Instead,
 dev builds search the `latest` Algolia index, which is based on the most recent
 _published_ version of Determined.
 
 That means dev builds of docs will never return quite the right results. Builds
 of unreleased docs will have out-of-date search results, while builds of old
-docs will have search results from the future.  Even so, because the Sphinx
+docs will have search results from the future. Even so, because the Sphinx
 search results are so bad, this is a tradeoff we are willing to accept.
 
 ### Canonical URLs
 
-Our site configures canonical links to point to `/latest` all the time.  This
+Our site configures canonical links to point to `/latest` all the time. This
 is necessary for optimizing SEO, and is a common practice on other docs sites
-(python standard library docs, for instance).  As a result, the Algolia crawler
+(python standard library docs, for instance). As a result, the Algolia crawler
 must be configured with `ignoreCanonicalTo: true` before it will index anything
 other than `/latest`.
+
+### Theme and customization
+
+Currently using a customized version of `sphinx-book-theme`.
```

Comment:
We've customized the `sphinx-book-theme` to suit our needs.

Replaces: `Currently using a customized version of `sphinx-book-theme`.`